### PR TITLE
[PLATFORM-1237]: Minimize oci image size

### PR DESCRIPTION
**Pull request by cottinisimone**
https://prima-assicurazioni-spa.myjetbrains.com/youtrack/issue/PLATFORM-1237

At the end this is the final size of the built image. If you have other suggestions feel free to propose :).

```
REPOSITORY                             TAG                IMAGE ID       CREATED         SIZE
localauth0                             latest             f9f25d609e26   5 minutes ago   8.94MB
```

### CODE REVIEW

**Comment by cpiemontese** [comment]
This is really cool (I'll leave a proper review to someone else because I don't feel qualified)

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1 @@
+target
```
Comment (MaeIsBad) [inline .dockerignore:1]: We don't need this right? Shouldn't docker use .gitignore by default?

**Review by MaeIsBad — COMMENTED** [review]
Everything seems great. One small question but I think we can merge this :rocket:

File: .dockerignore (hunk 1 above)
Comment (cottinisimone) [reply .dockerignore:1]: It seems not since it was copied in the context while building the image

**Review by MaeIsBad — APPROVED** [review]

**Pull request merged**